common/src/main/java/com/wynntils/mc/mixin/PlayerRendererMixin.java

```diff
@@ -60,4 +61,25 @@ private void onNameTagRenderPre(
             ci.cancel();
         }
     }
+
+    @Inject(
+            method =
+                    "render(Lnet/minecraft/client/player/AbstractClientPlayer;FFLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;I)V",
+            at = @At("RETURN"),
+            cancellable = true)
+    private void onRenderPost(
+            AbstractClientPlayer entity,
+            float entityYaw,
+            float partialTicks,
+            PoseStack matrixStack,
+            MultiBufferSource buffer,
+            int packedLight,
+            CallbackInfo ci) {
+        PlayerRenderEvent event =
+                new PlayerRenderEvent(entity, entityYaw, partialTicks, matrixStack, buffer, packedLight);
+        MixinHelper.post(event);
+        if (event.isCanceled()) {
```

Comment:
There is never any point in making a "post" mixin cancellable, since the method has already done everything it should. So you can just remove the `cancellable = true` from the annotation, and this code.

(Also, if you really wanted to make this event cancellable, you'd have needed to make the new event `@Cancellable` as well)